nienaruszalnosc-danych.rst

```diff
@@ -139,6 +138,19 @@ System KSI pozwala na udowodnienie, że określony stan bazy danych istniał w k
 2. Odwoływanie się do łańcucha sum kontrolnych w podpisach elektronicznych. Dzięki temu podpis uczciwego człowieka składany na dokumencie w danej sprawie poświadczałby również historię tej sprawy i uniemożliwiał jej zmianę do tego punktu.
 3. Publikację cząstkowych sum kontrolnych w inny sposób (np. publikacja sumy konrolnej sprawy w powiadomieniach emailowych wysyłanych obywatelowi).
 
+Długoterminowa archiwizacja podpisów
+------------------------------------
+
+Tradycyjna metoda konserwacji podpisu polega na opatrywaniu go pieczęcią elektroniczną poświadczającą, że podpis został utworzony przed określonym punktem czasowym (w którym tylko właściwa osoba dysponowała danymi pozwalającymi na złożenie tego podpisu). Pieczęć korzysta przy tym z tej samej technologii, co sam podpis. Zwiększenie mocy obliczeniowej komputerów niesie ze sobą ryzyko złamania zarówno jednego, jak i drugiego. Przyjmuje się, że wejście w życie komputerów kwantowych będzie oznaczało koniec współcześnie popularnych metod podpisywania (por. dokument “`CNSA Suite and Quantum Computing FAQ`_”). Za pomocą komputerów kwantowych będzie można, mając dostęp do publicznych certyfikatów, podrobić zarówno podpis pod dokumentem, jak i pieczęć znakującą ten podpis czasem.
```

Comment:
jestem za wspomnieniem o komputerach kwantowych, ponieważ mamy tutaj do czynienia nie tylko ze zwiększeniem mocy obliczeniowej, ale też ze zmianami w kwestii złożoności obliczeniowej niektórych problemów, więc to trochę inna skala